docs/content/anonymization/command.md

```diff
@@ -0,0 +1,119 @@
+---
+outline:
+   - 2
+   - 3
+---
+
+# Anonymization command
+
+Considering your anonymization has been configured, you can
+now anonymize a backup file running:
+
+```sh
+console db-tools:anonymize path/to/your/backup/to/anonymized
+```
+
+This command will successively:
+
+1. Backup your current database,
+2. Restore the given backup file,
+3. Anonymize the data from the given backup file,
+4. Backup the newly anonymized database, **overwritting the given backup file**,
+5. Restore your database to its original state from the backup done in step 1.
+
+::: warning
+The `db-tools:anonymize` command alone is not enought to ensure you follow GDPR best practices.
+It depends on:
+
+* How you correctly configured your anonymization (obviously),
+* Where you run this command: you are going to anonymized a backup file,
+  this backup file contains sensitive data. Thus, following GDPR recommendations
+  this **backup file should never transit on an unsecured environment**.
+
+Read the next section to learn more about a proper workflow.
+:::
+
+## A GDPR-friendly workflow
+
+Here is an exemple of workflow - that follows GDPR recommendations - to retrieve anonymized production
+data on your local environement.
+
+### Hypotheses
+
+* You have 2 secured environments : *production* and *preproduction*,
```

Comment:
... and you can securely copy files from one to another,

Reply:
It could be any environment, not even preproduction, all it needs is the CLI to work and a database, it doesn't even need to be a complete working env. This might be nice to add as a tip under ?

Reply:
For what it worth, you could even dump and restore the production database in another schema or database directly from the production CLI, then run the GDPR workflow by enforcing the new database using an environment variable:

```
bin/console ...:backup
DATABASE_URL="vendor://safe_not_prod/database" bin/console ...:restore
DATABASE_URL="vendor://safe_not_prod/database" bin/console ...:anonymize:run
```

Reply:
For the last comment, I agree, but do we want to intensively use the database server to anonymize in production?

(I guess you'll tell me the `DATABASE_URL="vendor://safe_not_prod/database"` could be on another database server...)